src/components/SelectFormField/styles.module.scss

```
@@ -1,70 +1,90 @@
+    font-family: var(--m3-body-font-family) !important;
+    font-size: 16px !important;
+    letter-spacing: 0.5px;
+}
 
-        .ant-form-item-explain-error {
-            font-size: 12px;
-            font-weight: 400;
-            line-height: 16px;
-        }
-    }
+.select :global(.ant-form-item-explain-error) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)